tests/conftest.py

```diff
@@ -308,13 +304,24 @@ def docker_image(boto_model_file):
         else:
             raise
 
-    tag = '{}/{}:{}'.format(registry, repository_name, '1.0.0')[8:]
-
     # pull existing image for layer cache
     try:
         client.images.pull(tag, auth_config={'username': username, 'password': password})
+        # the image with this tag already exists
```

Comment:
should we get the image from pull above, and actually check if this image exists? and then return the tag?

Reply:
the tag is part of the pull request, if there isn't a not found exception we can safely assume the image exists with the tag we provided

Reply:
Sounds good to me